README.md

```diff
@@ -86,6 +86,20 @@ Currently, supported relations are:
 - `loki-config`, for interfacing with [loki][Loki operator].
 - `grafana-dashboards`, for interfacing with [grafana][Grafana operator].
 
+
+### About Juju Topology
+
+This charm forwards alert rules, recording rules and dashboards but does not add its own metadata to the topology.
+
+The [Juju topology](https://charmhub.io/observability-libs/libraries/juju_topology) describes a node in the model, not the data flow.
+
+For this charm we don’t want this as it:
+
+1. Increases cardinality.
+2. Points at the wrong node in the model.
```

Comment:
I think we should clearly state here why we do not want to mark rules with their origin being cos-config.

For example:

> Since a cos-config _app_ needs to be deployed per app, topology labels _could_ be used to give a since of origin (as in data flow). However, this would be semantically inconsistent with charms, where topology labels indicate a node in the model. This may unpredictably interfere with `group_by` directives and ...

What other reasons?